src/fragmentarium/domain/fragment.ts

```diff
@@ -78,6 +78,36 @@ export interface ScriptDto {
 
 export class Fragment {
   readonly [immerable] = true
+  museumKey:
```

Comment:
The valid keys are already covered by `MuseumKey` from museum.ts, so delete this.